README.md

```diff
@@ -1,8 +1,4 @@
-<p align="center">
-  <a href="https://github.com/awslabs/aws-sam-local" align="center">
-    <img src="media/sam-banner.png" alt="SAM Extended logo" title="SAM" align="center" />
-  </a>
-</p>
+# ![Logo](media/sam-local-banner.png)
```

Comment:
Can you retain the HTML tag and just change image URL? It will keep the image aligned to center of README